documentation/API_documentation/DedicatedNetworks_GeneralDescription.md

```diff
@@ -101,13 +106,13 @@ sequenceDiagram
     participant Network as Physical Network
     participant D as Device(s)
     Note over App,D: Pre-requisites completed
-    rect lightcyan
-        note right of App: Selection of Profile needed for Dedicated Network
+    rect black
+        note right of App: 1: Reading Profiles
         App->>P: GET /profiles
         P->>App: 200 OK Profiles [profileId, ...]
     end
-    rect lightcyan
-        note right of App: Creation of a Dedicated Network
+    rect black
```

Comment:
I don't think, that the result is intended:

<img width="1206" height="674" alt="Image" src="https://github.com/user-attachments/assets/0796ec51-5800-4f56-b8c7-e411e8196e64" />